### DISCUSSION TITLE
What browser?

### DISCUSSION THREAD

**Discussion by APerricone** [cssbattle/forum#237] [Q&A]
Hello, I am spending some fun time on CssBattle and I see this strange behaviour on [open battle #100](https://cssbattle.dev/play/100). If I try the leading solution:
![image](https://user-images.githubusercontent.com/22457168/159109902-d87870dd-99d6-4bf9-9413-9256458d3f53.png)
On my Google Chrome 99.0.4844.74 It is rendered completly wrong but I press submit It gives me 100%, aside it is scorrect that allow to load someone else solution and stats with it, **What browser I should use for the correct solution?** for example on battle 103 I have the opposite problem, It looks pretty the same and the engine gives me the 99.9%:
![image](https://user-images.githubusercontent.com/22457168/159110101-4540fc5a-3899-46f1-b13d-39bc531a7019.png)

**Comment by shuffledbytes** [comment]
> Submitted code solutions are rendered in a headless Chromium browser on a Ubuntu 18.04 server.
> 
> Your code is rendered as it is without any modification and the browser settings are all set on default.
> 
> There might be slight differences in how different browsers render your code in the game preview. Hence till the time we find a way a reliable way to normalize these differences, we recommend playing on Chrome, Chromium, Brave or Safari browser (these are the ones we have tested).

**Reply by APerricone** [reply]
sorry to insist,As I said I tested on my Chrome  99.0.4844.74, to be sure I tested also on Chromium 102.0.4953.0, on Firefox 98.0.1 and on Edge  99.0.1150.46, all of them render it in a wrong way but If you submit it the result is a 100% match

**Comment by gunnarbittersmann** [comment]
Same reason as discussed in #231. Rename `--` to `--a` or similar.

**Reply by APerricone** [reply]
I think that the best solution is put somewhere on the site, may be on about page, the version of the rendering engine used, for me, I can go on [this page](https://www.chromium.org/getting-involved/download-chromium/#downloading-old-builds-of-chrome-chromium) and download a compatible version
